system-variables.md

```diff
@@ -463,7 +479,7 @@ MPP is a distributed computing framework provided by the TiFlash engine, which a
 ### tidb_backoff_lock_fast
 
 - Scope: SESSION | GLOBAL
-- Default value: `100`
+- Default value: `10`
```

Comment:
@seiya-annie PTAL this default value change. From which version this value has been changed?

Reply:
it's still 100 in v5.3.0, maybe a new change for master

Reply:
In the checklist at the time there was no option, but I selected master.

This affects TiDB 5.4 and above.